workflow-templates/gradle-check/action.yml

```diff
@@ -0,0 +1,41 @@
+name: gradle check
+description: Run gradle check
+
+inputs:
+  jdk-versions:
+    description: comma separated list of JDK versions-distros to test against
+    required: true
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+    - name: JDK8-zulu
+      if: contains('${{ inputs.jdk-versions }}', '8-zulu')
+      uses: actions/setup-java@v4
+      with:
+        java-version: 8
+        distribution: 'zulu'
+    - name: JDK8-temurin
+      if: contains('${{ inputs.jdk-versions }}', '8-temurin')
+      uses: actions/setup-java@v4
+      with:
+        java-version: 8
+        distribution: 'temurin'
+    - name: JDK11-temurin
+      if: contains('${{ inputs.jdk-versions }}', '11-temurin')
+      uses: actions/setup-java@v4
+      with:
+        java-version: 11
+        distribution: 'temurin'
+    - name: JDK17-temurin
+      if: contains('${{ inputs.jdk-versions }}', '17-temurin')
+      uses: actions/setup-java@v4
+      with:
+        java-version: 17
+        distribution: 'temurin'
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v4
+    - name: Check with Gradle Wrapper
```

Comment:
I am planning to add `gradle test` too, because when there are API changes `check` which depends on `apiCheck` fails, on such an occasion `test` can provide some information. See https://github.com/huanshankeji/exposed-vertx-sql-client/commit/cbf669241e8073764ab852aacc27caf904a64214.

Reply:
OK. `test` can be run in the same runner as `check` right?

Reply:
I think there is no problem.